Automattic/woocommerce-payments
# Update task-list to use the TIME.DAY_IN_MS from constants file

Fixes #5843

#### Changes proposed in this Pull Request

This PR replaces `DAY_IN_MS` in the `client/overview/task-list/index.js` file with the `TIME.DAY_IN_MS` constant from `client/constants.ts`.





This PR consolidates usage of a "day in milliseconds" variable to the constant defined in the constants file. This constant is also currently in use in the Fraud and Risk Tools [discoverability banner](https://github.com/Automattic/woocommerce-payments/blob/develop/client/components/fraud-risk-tools-banner/index.js#L38).





#### Testing instructions





* Check out this branch.
* All tests should pass.
* Make a purchase with the dispute card: `4000000000000259`.
* Make a hard refresh on the WCPay Overview page to see the dispute task show up. 
* Click the three dots menu on the right side of the task and click, "Remind me later".
* You should see a notice appear in the lower left that says: "Task postponed until tomorrow Undo"
* Then run this command to get the remind later timestamp: `npm run wp option get woocommerce_remind_me_later_todo_tasks`
* Check that the timestamp is one day in the future.

-------------------


**Post merge**
